ivi/rohdeschwarz/rohdeschwarzBaseScope.py

```diff
@@ -83,6 +83,15 @@
         'positive': 'pos',
         'negative': 'neg',
         'either': 'eith'}
+SearchConditionMapping = {
```

Comment:
I think these should be in the MeasurementFunctionMapping

Reply:
Fixed